### ISSUE TITLE
multi-class annotation

### ISSUE THREAD

**Issue by apeck12** [nextpyp/cet_pick#13]
Hi developers, 
I wondered if I could get some guidance on running MiloPYP for multi-class particle prediction. I have annotations for five particle classes and planned to run the following command based on this [tutorial](https://nextpyp.app/milopyp/quick_tutorial/#3d-interactive-session):
```
python /hpc/projects/krios1.processing/software/milopyp/cet_pick/cet_pick/main.py semi --down_ratio 2 --num_epochs 10 --bbox 16 --exp_id sample_refinement --dataset semi --arch unet_5 --save_all --debug 4 --val_interval 1 --thresh 0.85 --cr_weight 0.1 --temp 0.07 --tau 0.01 --lr 5e-4 --train_img_txt train_image.txt --train_coord_txt training_coordinates.txt --K 900 --compress --order xyz --gauss 0.8 --contrastive --last_k 3
```
Is there a command line argument to specify the number of classes? From [this](https://github.com/nextpyp/cet_pick/blob/main/cet_pick/opts.py#L314) it seems that the default number of classes is 1 in `semi` mode, or does that get updated based on the number of classes specified in the training_coordinates.txt file? Any advice would be appreciated, and sorry if I missed the documentation on how to do this correctly. 
Thanks,
Ariana

**Comment by abartesaghi** [comment]
Ariana,
You didn't miss anything, there is no support for multiple classes during MiLoPYP's refinement stage. You will need to train a different model for each species using the corresponding set of labels.
Alberto

**Comment by apeck12** [comment]
Thanks so much for that clarification.
